src/integration/setup.ts

```diff
@@ -0,0 +1,108 @@
+import { IndexList, Pinecone } from '../index';
+import {
+  diffPrefix,
+  generateRecords,
+  globalNamespaceOne,
+  prefix,
+  serverlessIndexName,
```

Comment:
I mentioned this in another comment, but I really don't think we should be using a hard-coded value for the index name, I think it should be a randomized and specific to a single test run, so across all files.